src/pynwb/icephys.py

```diff
@@ -71,7 +71,7 @@ class PatchClampSeries(TimeSeries):
              'doc': 'IntracellularElectrode group that describes the electrode that was used to apply '
                      'or record this data.'},
             {'name': 'gain', 'type': 'float', 'doc': 'Units: Volt/Amp (v-clamp) or Volt/Volt (c-clamp)'},  # required
-            {'name': 'stimulus_description', 'type': str, 'doc': 'the stimulus name/protocol', 'default': "NA"},
+            {'name': 'stimulus_description', 'type': str, 'doc': 'the stimulus name/protocol', 'default': "N/A"},
```

Comment:
```suggestion
            {'name': 'stimulus_description', 'type': str, 'doc': 'the stimulus name/protocol. Deprecated as of NWB 2.3.0. Set to N/A by default.', 'default': "N/A"},
```

Reply:
It is deprecated only on `IZeroClampSeries`. I'll update the docstring for that.